src/semgrep_agent/utils.py

```diff
@@ -63,7 +63,10 @@ def maybe_print_debug_info(meta: "GitMeta") -> None:
 
 def render_error(error: Mapping[str, Any]) -> Sequence[str]:
     spans = error.get("spans")
-    msg = error.get("long_msg", "")
+    msg: str = error.get("long_msg", error.get("message", ""))
+    newline_ix = msg.find("\n")
+    if newline_ix > -1:
+        msg = msg[: newline_ix - 1]
```

Comment:
Why are we doing this modification?

Reply:
b/c otherwise we'd dump enormous but useless stack traces to the console

Reply:
Can you add an inline comment mentioning that

Reply:
To help know what needs moving later when we port to semgrep-cli